R/layers.R

```diff
@@ -715,3 +729,61 @@ removeGeoJSON = function(map, layerId) {
 clearGeoJSON = function(map) {
   invokeMethod(map, NULL, 'clearGeoJSON')
 }
+
+#' Add UI controls to switch layers on and off
+#'
+#' Uses Leaflet's built-in
+#' \href{http://leafletjs.com/reference.html#control-layers}{layers control}
+#' feature to allow users to choose one of several base layers, and to choose
+#' any number of overlay layers to view.
+#'
+#' @param map the map to add the layers control to
+#' @param baseGroups character vector where each element is the name of a group.
+#'   The user will be able to choose one base group (only) at a time. This is
+#'   most commonly used for mostly-opaque tile layers.
+#' @param overlayGroups character vector where each element is the name of a
+#'   group. The user can turn each overlay group on or off independently.
+#' @param position position of control: 'topleft', 'topright', 'bottomleft', or
+#'   'bottomright'
+#' @param options a list of additional options, intended to be provided by
+#'   a call to \code{layersControlOptions}
+#'
+#' @examples
+#' \donttest{
```

Comment:
CRAN may ask us to explain why `\donttest{}`. I guess it is fine not to use `\donttest{}` here, since I don't see how it can be problematic. Or we add examples to the website, and provide a link to the web page here.